client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -51,6 +51,31 @@ export default React.createClass( {
 		return this.props.domains || [ this.props.domain ]
 	},
 
+	wrongNSMappedDomains() {
+		debug( 'Rendering wrongNSMappedDomains' );
+		const wrongMappedDomains = this.getDomains().filter( domain =>
+			domain.type === domainTypes.MAPPED && ! domain.hasZone );
+
+		let learnMoreLink = ( <a
+			href="https://support.wordpress.com/domains/change-name-servers"
```

Comment:
I am not quite sure this is a proper link...

The whole notice should be displayed, when domain is bought elsewhere and Name Servers should be set in the admin panel of domain provider.
The attached link shows how to point domain bought on wp.com to outside name servers.
I believe this is the link that was supposed to be here: https://en.support.wordpress.com/map-existing-domain/#2-ask-your-domain-provider-to-update-your-dns-settings

Also, we show the same link in case of subdomain, in which case we should give this link (I believe): https://en.support.wordpress.com/map-subdomain/#instructions-for-mapping-subdomains
And information about CNAME....

Also I really find the Domain Helper output being pretty helpful and a bit more in context then a general support page.
```
'https://en.support.wordpress.com/domain-helper/?host=' + wrongMappedDomains[0].name
```